xl2times/utils.py

```diff
@@ -186,22 +186,17 @@ def create_regexp(pattern):
         pattern = remove_negative_patterns(pattern)
     if len(pattern) == 0:
         return re.compile(pattern)  # matches everything
-    # escape special characters
-    # Backslash must come first
-    special = "\\.|^$+()[]{}"
-    for c in special:
-        pattern = pattern.replace(c, "\\" + c)
```

Comment:
We don't need this any more?

Reply:
I believe we don't. Removing it didn't cause any regressions or errors...

Reply:
I'm wondering if we should add a warning/error if we find any special characters in the pattern? If someone uses a model that uses a special character, the failure might be silent / hard to debug.